src/extension.ts

```diff
@@ -159,6 +163,44 @@ export function activate(context: vscode.ExtensionContext): void {
     void vscode.commands.executeCommand('setContext', 'titleCommandsRegistered', true);
 }
 
+export async function attachToCurrentDebugTarget(
+    context: vscode.ExtensionContext): Promise<void> {
+    if (!telemetryReporter) {
+        telemetryReporter = createTelemetryReporter(context);
+    }
+
+    telemetryReporter.sendTelemetryEvent('command/attachToCurrentDebugTarget');
+
+    // Attempt to attach to active CDP target
+    const session = vscode.debug.activeDebugSession;
+    if (!session) {
+        vscode.window.showErrorMessage('No active debug session');
+        return;
+    }
+
+    let targetWebsocketUrl;
+    try {
+        const addr: any = await vscode.commands.executeCommand(
+        'extension.js-debug.requestCDPProxy',
+        session.id,
+        );
+        const formed = `ws://${addr.host}:${addr.port}${addr.path || ''}`;
+        targetWebsocketUrl = formed;
+    } catch (e) {
+        vscode.window.showErrorMessage(e.message);
```

Comment:
We don't necessarily need to do this in this PR, but we should start utilizing `sendTelemetryErrorEvent` to record failures that happen within the extension code.  Currently, we only use `sendTelemetryErrorEvent` for DevTools errors.

Feel free to send error telemetry in the places where you show the error toast notification, but otherwise we can change this when we make a sweep of the extension code to implement better error telemetry.